14/umbraco-commerce/installation/install.md

```diff
@@ -96,6 +96,10 @@ For most sites using a single solution, the above will be all you need to instal
    </tbody>
 </table>
 
+{% hint style="info" %}
+If you encounter an SQLite error after installing Umbraco Commerce, you may need to enable SQLite support. For more information, see the [Configure SQLite support](https://docs.umbraco.com/umbraco-commerce/how-to-guides/configure-sqlite-support) article.
```

Comment:
Wouldn't it be better to use a relate link instead? So, something like `../../installation/install` instead.

Reply:
Done